boxes/blank/README.md

```diff
@@ -1,76 +1,67 @@
-              |— src
-                     | The Noir smart contract source files are here.
-                     |— main.nr - the cloned noir contract, your starting point
-                     |- interface.nr - autogenerated from main.nr when you compile
-               |— Nargo.toml [Noir build file, includes Aztec smart contract dependencies]
-       |— artifacts
-              |  These are both generated from `contracts/` by the compile command
-              |— blank_contract.json
-              |— blank.ts
-       |— tests
-              | A simple end2end test deploying and testing the minimal contract on a local sandbox
-              | using the front end helper methods in index.ts
-              | The test requires the sandbox and anvil to be running (yarn start:sandbox).
-              |- blank.contract.test.ts
-```
+Unbox the box with:
 
-Most relevant to you is likely `src/contracts/main.nr` (and the build config `src/contracts/Nargo.toml`). This contains the example blank contract logic that the frontend interacts with and is a good place to start writing Noir.
+```bash
+aztec-cli unbox blank
+```
 
-The `src/artifacts` folder can be re-generated from the command line
+and install dependencies:
 
 ```bash
-yarn compile
+yarn
 ```
 
-This will generate a [contract artifact](src/artifacts/test_contract.json) and TypeScript class for the [Aztec smart contract](src/contracts/main.nr), which the frontend uses to generate the UI.
+This sandbox requires [Docker](https://www.docker.com/) to be installed _and running_ locally. In the event the image needs updating, you can run `yarn install:sandbox` (see [sandbox docs](https://docs.aztec.network/developers/cli/main) for more information.)
 
-Note: the `compile` command seems to generate a Typescript file which needs a single change -
+Time to build. Run:
 
+```bash
+yarn start
 ```
-import TestContractArtifactJson from 'text_contract.json' assert { type: 'json' };
-// need to update the relative import to
-import TestContractArtifactJson from './test_contract.json' assert { type: 'json' };
-```
 
-After compiling, you can re-deploy the updated noir smart contract from the web UI. The function interaction forms are generated from parsing the contract artifact, so they should update automatically after you recompile.
+Your React app is waiting for you on port 5176. Time to make it your own!
+
+In the `src/contracts` folder, you'll find the default contract being deployed. Don't forget to recompile with `aztec-nargo compile`! Read the [aztec.nr documentation](https://docs.aztec.network/dev_docs/contracts/main) to get started with the `aztec.nr` framework.
+
+[Read the full Sandbox reference](https://docs.aztec.network/dev_docs/cli/sandbox-reference) for more info on what exactly is happening on your machine!
 
-## Learn More
+## More info
 
-To learn more about Noir Smart Contract development, take a look at the following resources:
+Here's what is inside your `src` folder:
```

Comment:
Looks like the following needs some updating as well.

Reply:
I'll just remove it like in the other README